lib/azure/role.rb

```diff
@@ -368,7 +368,7 @@ def setup(params)
                   xml.Port params[:port]
                   xml.Protocol 'TCP'
                 }
-                elsif params[:os_type] == 'Linux' and params[:bootstrap_proto].downcase == 'ssh'
+                else
```

Comment:
Recently we made changes to not to add any endpoint in case of `cloud-api` bootstrap protocol. I think this change will affect this.